index.html

```diff
@@ -0,0 +1,122 @@
+    <h1 class="titleh1">PREPARE FOR THE FUTURE</h1>
+    <h1 class="year">2015</h1>
+    <div class="intro-text">
+      <p>Learn JS with focus on user interfaces and DOM manipulation. We will show you how to manipulate Dom elements in the right way.</p>
+    </div>
+    <div class="dates-container">
+      <h2 class="dates">2015.10.15(THU)~16(FRI)</h2>
+      <h4 class="place">@ Informatica Technical Institute, Santa Cruz, Bolivia</h4>
+    </div>
+  </section>
+  <section class="program">
+    <h3 class="main-program">Main Program</h3>
+    <hr class="underline">
+    <div class="card-container">
+      <div class="card">
+        <img src="images/conference-icon.png" alt="Conference Icon" class="img-cards">
+        <h5 class="card-title">Lecture</h5>
+        <p class="card-text">Listen to lectures from speakers from around the world and learn about the latest trends in the world.
+        </p>
+      </div>
+      <div class="card">
+        <img src="images/exhibition-icon.png" alt="Exhibition Icon" class="img-cards">
+        <h5 class="card-title">CC exhibition</h5>
+        <p class="card-text">Let's meet the creations of artists from various fields who share the open spirit of CC.
+        </p>
+      </div>
+      <div class="card">
+        <img src="images/forum-icon.png" alt="Forum Icon" class="img-cards">
+        <h5 class="card-title">Forum</h5>
+        <p class="card-text">We have time to share our thoughts and opinions with experts by topic.</p>
+      </div>
+      <div class="card">
+        <img src="images/workshop-icon.png" alt="Workshop Icon" class="img-cards">
+        <h5 class="card-title">Workshop</h5>
+        <p class="card-text">You can create your own creations using open source instead of just looking at them.</p>
+      </div>
+      <div class="card">
+        <img src="images/party-icon.png" alt="Party Icon" class="img-cards">
+        <h5 class="card-title">CC Ignite</h5>
+        <p class="card-text">Create opportunities to freely network with CC personnel from around the world.</p>
+      </div>
+    </div>
+    <div class="footer-card">
+      <a href="#">View full program</a>
+    </div>
+  </section>
+   <!--This section was build with JavaScript-->
+  <section class="exhib-sections">
+    <h3 class="title">Our Team</h3>
+    <hr class="big-title-bar" />
+    <ul class="exhib-section-list"></ul>
+  </section>
+  <section class="partners">
+    <h3 class="our-partners">Our Partners</h3>
+    <hr class="underline">
+    <div class="logo-container">
+      <a class="sponsor-link" href="https://www.google.com/" target="_blank" rel="noopener noreferrer">GOOGLE</a>
+      <a class="sponsor-link" href="https://www.asus.com/" target="_blank" rel="noopener noreferrer">ASUS</a>
+      <a class="sponsor-link" href="https://www.ebay.com/" target="_blank" rel="noopener noreferrer">EBAY</a>
+      <a class="sponsor-link" href="https://www.oracle.org/" target="_blank" rel="noopener noreferrer">ORACLE</a>
+    </div>
```

Comment:
- [ ] Kindly make sure the partners' logos are well aligned at the centre of the page, as seen in the original design👌

![footer-cap](https://user-images.githubusercontent.com/44624138/143657272-10c3d63f-16d8-461f-a40f-8eb5437b9ac4.png)